src/editor/Core/ConfigurationDefaults.re

```diff
@@ -10,30 +10,26 @@ let getDefaultConfigString = configName =>
     Some(
       {|
 {
-  "editor.minimap.enabled": true,
-  "editor.insertSpaces": false,
+  "editor.detectIndentation": true,
+  "editor.largeFileOptimizations": true,
+  "editor.highlightActiveIndentGuide": true,
   "editor.indentSize": 4,
-  "editor.tabSize": 4,
-  "editor.DetectIndentation": true,
-  "editor.LargeFileOptimizations": true,
+  "editor.insertSpaces": false,
+  "editor.lineNumbers": "on",
   "editor.matchBrackets": true,
   "editor.minimap.enabled": true,
-  "editor.minimap.showSlider": true,
   "editor.minimap.maxColumn": 120,
-  "editor.lineNumbers": "on",
-  "editor.insertSpaces": false,
-  "editor.indentSize": 4,
-  "editor.tabSize": 4,
+  "editor.minimap.showSlider": true,
   "editor.renderIndentGuides": true,
-  "editor.highlightActiveIndentGuide": true,
   "editor.renderWhitespace": "all",
+  "editor.tabSize": 4,
+  "editor.zenMode.hideTabs": true,
+  "files.exclude": ["_esy", "node_modules"],
   "workbench.activityBar.visible": true,
   "workbench.editor.showTabs": true,
+  "workbench.iconTheme": "vs-seti",
   "workbench.sideBar.visible": true,
   "workbench.statusBar.visible": true,
```

Comment:
The parser we are using doesn't like/we've not set it up to like a trailing comma, so I'm assuming this is what is causing the CI failure.